design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,147 @@
+type Mutation {
+    createNegotiableQuote(
+        cart_id: ID! @doc(description: "Cart ID")
+        quote_name: String @doc(description: "Quote name")
+        comment: String @doc(description: "Comment")
+        files: [FileInput!] @doc(description: "Attached files")
```

Comment:
In another parts of the schema this field is called `attachments`. It is better to be consistent.

```suggestion
        attachments: [FileInput!] @doc(description: "Attached files")
```

Reply:
We should skip attachments on this iteration of the schema, and follow-on with a separate PR. We haven't yet come to a decision internally, but it looks like we're learning towards file uploads _not_ passing through the GraphQL layer